src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -0,0 +1,1110 @@
+                                        options={this.geneOptions}
+                                        clearable={false}
+                                        searchable={false}
+                                    />
+                                </div>
+                                <div style={{marginLeft:7, display:"inline"}}>
+                                    <DefaultTooltip
+                                        placement="right"
+                                        overlay={<span>{this.geneLock ? "Gene selection synchronized in both axes" : "Gene selections independent in each axis"}</span>}
+                                    >
+                                        <div>
+                                            <LockIcon locked={this.geneLock} className="lockIcon" onClick={vertical ? this.onVerticalAxisGeneLockClick : this.onHorizontalAxisGeneLockClick}/>
+                                        </div>
+                                    </DefaultTooltip>
+                                </div>
+                            </div>
+                        </div>
+                        <div>
+                            Profile Type
+                            <ReactSelect
+                                name={`${vertical ? "v" : "h"}-profile-type-selector`}
+                                value={axisSelection.molecularProfileType}
+                                onChange={vertical ? this.onVerticalAxisProfileTypeSelect : this.onHorizontalAxisProfileTypeSelect}
+                                options={this.profileTypeOptions}
+                                clearable={false}
+                                searchable={false}
+                            />
+                        </div>
+                        <div>
+                            Profile Name
+                            <div style={{display:"flex", flexDirection:"row"}}>
+                                <div style={{width:"93%"}}>
+                                    <ReactSelect
+                                        name={`${vertical ? "v" : "h"}-profile-name-selector`}
+                                        value={axisSelection.molecularProfileId}
+                                        onChange={vertical ? this.onVerticalAxisProfileIdSelect : this.onHorizontalAxisProfileIdSelect}
+                                        options={this.profileNameOptionsByType[axisSelection.molecularProfileType+""] || []}
+                                        clearable={false}
+                                        searchable={false}
+                                    />
+                                </div>
+                                <InfoIcon
+                                    tooltip={<span>{vertical ? this.vertDescription : this.horzDescription}</span>}
+                                    tooltipPlacement="right"
+                                    style={{marginTop:"0.9em", marginLeft:5}}
```

Comment:
is there a reason units em?

Reply:
i guess theres no rational justification except it feels more semantic to me bc its sort of related to line height